.github/workflows/ci.yml

```diff
@@ -56,6 +56,12 @@ jobs:
         run: cargo doc --all --no-deps
       - name: Test with code coverage
         run: cargo tarpaulin --release --engine llvm --follow-exec --post-test-delay 10 --coveralls ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - name: Check Nix flake inputs
```

Comment:
Can this be added in a separate, parallel, job, within ci.yml? Also need to have a test for whether the nix-built `rip` works